addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -472,7 +472,7 @@ def grouping_key_generator(base_line, tax_values):
         # Compute values for invoice lines.
         line_extension_amount = 0.0
 
-        invoice_lines = invoice.invoice_line_ids.filtered(lambda line: line.display_type not in ('line_note', 'line_section'))
+        invoice_lines = invoice.invoice_line_ids.filtered(lambda line: line.display_type not in ('line_note', 'line_section') and line.product_id._check_edi_move())
```

Comment:
I'm wondering whether this check is needed here. Is the code below incorrect for combos?

I also wonder whether we should add the check here as well:
https://github.com/odoo/odoo/blob/9fed5ebe8f029e7de9f46b4b8bbc9056e672c1bf/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py#L307

(not sure though, need to double-check)

Reply:
> I'm wondering whether this check is needed here. Is the code below incorrect for combos?

I don't think the code below is incorrect but I don't think it's useful knowing combo is not allowed to have a tax set, and price should be 0 once added to the cart (in Pos) but I don't know enough about Accounting to be 100% sure it's not useful. 

All I know is that if the combo line is added to `invoice_line_vals_list` then we have this error coming back to us:
![image](https://github.com/user-attachments/assets/995b9a15-b19b-47c9-94dc-aaf2e26b6125)
So we still have the issue at the end right?

> I also wonder whether we should add the check here as well:

Maybe... I don't know enough about accounting to know how to test that. Do you know of any workflow that I could use to do so?

> Thanks for your fix! Could you update the PR/commit message to include all modified modules? :)
Commit was already updated but now both PR and commit are.

> Could you update the PR/commit message to include all modified modules?

Oops I had modified the commit message but forgot the PR, done !